homeassistant/components/cert_expiry/sensor.py

```diff
@@ -83,6 +83,7 @@ def extra_state_attributes(self):
         return {
             "is_valid": self.coordinator.is_cert_valid,
             "error": str(self.coordinator.cert_error),
+            "days_remaining": int(self.coordinator.days_remaining),
```

Comment:
This sensor provides the expiry date, you can turn that into anything you like (e.g., the number of seconds until the expiry date).

Additionally, we don't allow these things to be put into attributes.